### PR TITLE
feat(positions): close position with max possible size

### LINKED ISSUE #4834: Make close position us max possible size
## The Chore

What we need to achieve and who for

## Tasks

- [ ] Confirm max possible size with core
- [ ] Use that for the close position transaction (rather than open volume) so that there is no chance of fills happening whilst the tx is going through

## Additional details / background info

https://vegaprotocol.slack.com/archives/C037STCKMGC/p1695104903200239

### PR DESCRIPTION
**Pull request by macqbat**
# Related issues 🔗

Closes #4834 

# Description ℹ️

Add max golang uint64 as size of closing position tx.

### CODE REVIEW

File: libs/positions/src/lib/positions-manager.tsx [hunk 1]
```diff
@@ -19,6 +18,8 @@ interface PositionsManagerProps {
   showClosed?: boolean;
 }
 
+const MAXINT64 = '9223372036854775807';
```
Comment (mattrussell36) [inline libs/positions/src/lib/positions-manager.tsx:21]: We use the same constant for deposit approvals. Perhaps we can add this to utils and share the value.

Can we also add some comments saying why we use this value

**Review by mattrussell36 — CHANGES_REQUESTED** [review]

File: libs/positions/src/lib/positions-manager.tsx (hunk 1 above)
Comment (macqbat) [reply libs/positions/src/lib/positions-manager.tsx:21]: Only we need there max 64 bit int for golang, and in deposit we have max 256 bit uint from ethereum lib, and I can't see there is int64 available. So I'll just move it to some constants file.

Comment (bwallacee) [reply libs/positions/src/lib/positions-manager.tsx:21]: is it worth getting a test in for deposit approval as well if we are building this into utils?

Comment (macqbat) [reply libs/positions/src/lib/positions-manager.tsx]: > is it worth getting a test in for deposit approval as well if we are building this into utils?

I think deposit will remain like it is now, with uint256 from ethereum and close position tx with int64 stored in the utilis lib

**Review by bwallacee — APPROVED** [review]

**Review by MadalinaRaicu — APPROVED** [review]

**Pull request merged**